FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib

```diff
@@ -115,7 +115,7 @@
                                 <fragment content="Your connection will not be secure while you update.">
                                     <attributes>
                                         <color key="NSColor" name="custom_grey50"/>
-                                        <font key="NSFont" size="13" name="Inter-Regular"/>
+                                        <font key="NSFont" metaFont="system"/>
```

Comment:
Why do we change font here?

Reply:
I've removed unnecessary paragraph style and set up correct fonts.